grassroots-backend/src/phone-canvass/Twilio.service.ts

```diff
@@ -110,28 +89,26 @@ export class TwilioService {
   async setSyncData(
     phoneCanvassId: string,
     data: PhoneCanvassSyncData,
-  ): Promise<DocumentInstance> {
+  ): Promise<void> {
     const TWILIO_SYNC_SERVICE_SID = getEnvStr(
       this.config,
       "TWILIO_SYNC_SERVICE_SID",
     );
 
-    const client = this.getClient();
+    const client = this.#getClient();
 
-    let doc: undefined | DocumentInstance;
     try {
       // This fails if the document doesn't already exist. Checking if it exists first introduces
       // an extra unnecessary round trip and some complexity.
       // Instead we just try to update it, and if that fails, create it instead.
-      doc = await client.sync.v1
+      await client.sync.v1
```

Comment:
We weren't using these return values, and they made writing mocks much more annoying.